app/src/main/java/edu/ucandroid/weathernotice/ui/main/MainViewModel.kt

```diff
@@ -22,7 +22,7 @@ class MainViewModel : ViewModel() {
     init{
         fetchLocations()
         fetchWeatherLocations("e")
-        firestore = FirebaseFirestore.getInstance()
+        //firestore = FirebaseFirestore.getInstance()
     }
```

Comment:
Commented out the "getInstance" method since we do not need to use this in the MVM class.

Reply:
Acknowledged.

Reply:
Committing commented out code is considered bad practice in version control, because version control offers better alternatives:
1) Use a feature branch for experimental features.
2) Delete what is not needed.  You can always compare with history to see what used to be there.
Commented out code is considered clutter in a version control system.